src/goto-symex/symex_dead.cpp

```diff
@@ -63,17 +63,5 @@ void goto_symext::symex_dead(statet &state)
     auto level2_it = state.level2.current_names.find(l1_identifier);
     if(level2_it != state.level2.current_names.end())
       symex_renaming_levelt::increase_counter(level2_it);
-
-    const bool record_events = state.record_events;
```

Comment:
Putting this reasoning into the code would be good.

Reply:
What are you referring to by "this reasoning?"